examples/swaps/swapV3.ts

```diff
@@ -0,0 +1,118 @@
+/**
+ * Example showing how to query and execute a v3 swap using a custom path
+ * Note that all v3 swaps require permit2 approvals
+ *
+ * Run with:
+ * pnpm example ./examples/swaps/swapV3.ts
+ */
+
+import {
+    Slippage,
+    SwapKind,
+    PERMIT2,
+    TokenAmount,
+    ChainId,
+    Permit2Helper,
+    SwapBuildCallInput,
+} from '../../src';
+import { TOKENS, approveSpenderOnToken } from 'test/lib/utils';
+import { setupExampleFork } from '../lib/setupExampleFork';
+import { queryCustomPath } from './queryCustomPath';
+import { parseUnits, Address } from 'viem';
+
+const swapV3 = async () => {
+    // Choose chain id to start fork
+    const chainId = ChainId.SEPOLIA;
+    const { client, rpcUrl, userAccount } = await setupExampleFork({ chainId });
+
+    // TODO: Fix query revert, see tenderly simulation: https://www.tdly.co/shared/simulation/f0b0a1de-4e88-4b8d-bf90-c25c4d2145ec
+    // Query swap results before sending transaction
+    const { swap, queryOutput } = await queryCustomPath({
+        rpcUrl,
+        chainId,
+        pools: ['0xA8c18CE5E987d7D82ccAccB93223e9bb5Df4A3c0'] as Address[], // https://test.balancer.fi/pools/sepolia/v3/0xa8c18ce5e987d7d82ccaccb93223e9bb5df4a3c0
```

Comment:
Still working on debugging this :sweat_smile: 

The pools listed on v3 FE are deploy 11 right?
https://test.balancer.fi/pools?networks=SEPOLIA&skip=0&textSearch=bal&protocolVersion=3